### PR TITLE
 Adding sample log statements in templates

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

 Adding sample log statements in templates

### Description of the Change

- Added sample log statement in all the service templates
- Changed forEach to for loop in logstreamers

### Benefits

No need to uncomment the log statement for checking logs

### CODE REVIEW

File: templates/api-template-go/main.go [hunk 1]
```diff
@@ -62,10 +62,12 @@ func Handler(ctx context.Context, event map[string]interface{}) (Response, error
 		var payLoad map[string]interface{}
 		// Your GET method should be handled here
 		if (event["method"].(string) == "GET"){
+      logger.VERBOSE("Sample log inside GET");
```
Comment (pspani) [inline templates/api-template-go/main.go]: change to info as default log level is info, ie only error, warning and info will be printed. Do in other files too

File: templates/lambda-template-nodejs/index.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ module.exports.handler = (event, context, cb) => {
       "configKeys": myVal
     };
 
+    logger.debug("Sample log for nodejs service");
```
Comment (pspani) [inline templates/lambda-template-nodejs/index.js]: change to info. make changes in other files too

File: templates/sls-app-template-go/functions/function1/main.go [hunk 3]
```diff
@@ -36,7 +36,7 @@ func Handler(ctx context.Context, event map[string]interface{}) (Response, error
 	 logger.VERBOSE("Generally speaking, most lines logged by your application should be written as verbose.");
 	 logger.DEBUG("Detailed information on the flow through the system.");
 	*/
-	logger.INFO("Interesting runtime events (Eg. connection established, data fetched etc.)")
+	logger.INFO("Sample log for function1")
```
Comment (pspani) [inline templates/sls-app-template-go/functions/function1/main.go]: keep the log statement. It gives context to the user

Comment (SiniWilson) [reply templates/sls-app-template-go/functions/function1/main.go]: Done

File: templates/lambda-template-nodejs/index.js (hunk 2 above)
Comment (SiniWilson) [reply templates/lambda-template-nodejs/index.js]: Done

File: templates/api-template-go/main.go (hunk 1 above)
Comment (SiniWilson) [reply templates/api-template-go/main.go]: Done

**Pull request merged**